info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T03:51:20.621Z"}
info: Connected to the database. {"timestamp":"2022-02-25T03:51:20.646Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T04:07:59.516Z"}
info: Connected to the database. {"timestamp":"2022-02-25T04:07:59.539Z"}
error: 
    status - 500 
    message - ValidationError: "Title" is required 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:10:47.279Z"}
error: 
    status - 500 
    message - ValidationError: "Title" is required 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:12:17.818Z"}
error: 
    status - 500 
    message - ValidationError: "Title" is required 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:13:08.709Z"}
error: 
    status - 500 
    message - ValidationError: "Title" is required 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:15:13.678Z"}
error: 
    status - 500 
    message - ValidationError: "Title" is required 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:16:19.333Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T04:16:55.388Z"}
info: Connected to the database. {"timestamp":"2022-02-25T04:16:55.413Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:63:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:88:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\index.js:281:22 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:17:03.072Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'split')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:46:54)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:88:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\index.js:341:12) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:19:48.974Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T04:22:15.509Z"}
info: Connected to the database. {"timestamp":"2022-02-25T04:22:15.541Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:22:27.460Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:22:34.289Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:63:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:88:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:22:46.896Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:63:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:88:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:23:15.648Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T04:24:04.393Z"}
info: Connected to the database. {"timestamp":"2022-02-25T04:24:04.416Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:24:12.533Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:24:14.254Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:24:22.280Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:24:23.178Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T04:24:31.336Z"}
info: Connected to the database. {"timestamp":"2022-02-25T04:24:31.362Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:24:34.641Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:24:40.193Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:24:55.944Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T04:26:48.816Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T04:26:52.585Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T04:28:15.570Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T04:28:19.638Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:28:27.085Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:34:00.812Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:38:57.369Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/note/ 
      method - GET 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-02-25T04:39:34.952Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:40:59.745Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:41:17.005Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:41:17.886Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:62:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:87:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T04:42:07.704Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T04:46:03.956Z"}
info: Connected to the database. {"timestamp":"2022-02-25T04:46:03.980Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:63:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:88:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T04:46:27.759Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T05:38:31.311Z"}
info: Connected to the database. {"timestamp":"2022-02-25T05:38:31.338Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:65:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:90:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:112:3) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T05:38:38.303Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T05:52:06.019Z"}
info: Connected to the database. {"timestamp":"2022-02-25T05:52:06.064Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'id')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:42:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at create (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:71:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:69:17
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:14
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T06:07:50.863Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T06:08:22.961Z"}
info: Connected to the database. {"timestamp":"2022-02-25T06:08:22.987Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'id')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:42:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at create (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:71:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:69:17
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:14
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T06:08:25.907Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'id')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:42:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at create (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:71:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:69:17
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:14
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T06:12:21.006Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T06:16:29.865Z"}
info: Connected to the database. {"timestamp":"2022-02-25T06:16:29.887Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T06:18:42.775Z"}
info: Connected to the database. {"timestamp":"2022-02-25T06:18:42.808Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T06:19:03.915Z"}
info: Connected to the database. {"timestamp":"2022-02-25T06:19:03.948Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'id')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:42:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at create (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:71:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:70:17
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:14
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T06:19:43.409Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T06:22:00.352Z"}
info: Connected to the database. {"timestamp":"2022-02-25T06:22:00.376Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'id')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:42:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at create (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:71:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:70:17
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:14
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T06:22:07.001Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'id')
    at _callee$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:42:45)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at create (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:71:17)
    at Layer.handle [as handle_request] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\express\lib\router\route.js:137:13)
    at C:\Users\anand\Desktop\fundoo-Notes\build\middlewares\auth.middleware.js:70:17
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:133:14
    at getSecret (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.module.exports [as verify] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\jsonwebtoken\verify.js:94:10) 
    url - /api/v1/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-25T06:22:09.695Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T06:26:58.482Z"}
info: Connected to the database. {"timestamp":"2022-02-25T06:26:58.512Z"}
error: 
      status - 400
      message - Authorization token is incorrect 
      url - /api/v1/note 
      method - POST 
      IP - ::1
     {"timestamp":"2022-02-25T06:27:06.476Z"}
error: 
      status - 400
      message - Authorization token is incorrect 
      url - /api/v1/note 
      method - POST 
      IP - ::1
     {"timestamp":"2022-02-25T06:27:11.229Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T08:53:52.794Z"}
info: Connected to the database. {"timestamp":"2022-02-25T08:53:52.819Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T08:58:29.744Z"}
info: Connected to the database. {"timestamp":"2022-02-25T08:58:29.767Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ ID: '62189adf0062220118ab9159' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/archive/62189adf0062220118ab9159 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:05:03.829Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T09:08:08.410Z"}
info: Connected to the database. {"timestamp":"2022-02-25T09:08:08.443Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: undefined }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/archive/62189adf0062220118ab9159 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:08:22.930Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: undefined }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/archive/62189adf0062220118ab9159 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:08:24.136Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T09:09:06.772Z"}
info: Connected to the database. {"timestamp":"2022-02-25T09:09:06.802Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T09:12:40.555Z"}
info: Connected to the database. {"timestamp":"2022-02-25T09:12:40.599Z"}
error: 
    status - 500 
    message - MongooseError: Callback must be a function, got [object Object]
    at Function.Model.$handleCallbackError (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:5067:11)
    at Function.Model.findByIdAndUpdate (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:2694:19)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:178:37)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.MakeArchive (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:201:18)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:251:32)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21) 
    url - /api/v1/note/archive/62189ce3aacd274b9041273f 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:12:57.311Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T09:13:48.115Z"}
info: Connected to the database. {"timestamp":"2022-02-25T09:13:48.148Z"}
error: 
    status - 500 
    message - MongooseError: Callback must be a function, got [object Object]
    at Function.Model.$handleCallbackError (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:5067:11)
    at Function.Model.findByIdAndUpdate (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:2694:19)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:178:37)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.MakeArchive (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:201:18)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:251:32)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21) 
    url - /api/v1/note/archive/62189ce3aacd274b9041273f 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:14:01.580Z"}
error: 
    status - 500 
    message - MongooseError: Callback must be a function, got [object Object]
    at Function.Model.$handleCallbackError (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:5067:11)
    at Function.Model.findByIdAndUpdate (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:2694:19)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:178:37)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.MakeArchive (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:201:18)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:251:32)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21) 
    url - /api/v1/note/archive/62189ce3aacd274b9041273f 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:14:02.474Z"}
error: 
    status - 500 
    message - MongooseError: Callback must be a function, got [object Object]
    at Function.Model.$handleCallbackError (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:5067:11)
    at Function.Model.findByIdAndUpdate (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\model.js:2694:19)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:178:37)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\anand\Desktop\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.MakeArchive (C:\Users\anand\Desktop\fundoo-Notes\build\services\note.service.js:201:18)
    at _callee6$ (C:\Users\anand\Desktop\fundoo-Notes\build\controllers\note.controller.js:251:32)
    at tryCatch (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\anand\Desktop\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21) 
    url - /api/v1/note/archive/62189ce3aacd274b9041273f 
    method - PUT 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-02-25T09:14:03.334Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T09:14:51.333Z"}
info: Connected to the database. {"timestamp":"2022-02-25T09:14:51.365Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T11:44:17.153Z"}
info: Connected to the database. {"timestamp":"2022-02-25T11:44:17.219Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T11:44:59.992Z"}
info: Connected to the database. {"timestamp":"2022-02-25T11:45:00.020Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T16:37:59.684Z"}
info: Connected to the database. {"timestamp":"2022-02-25T16:37:59.719Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T16:38:15.826Z"}
info: Connected to the database. {"timestamp":"2022-02-25T16:38:15.871Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ ID: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62189a580062220118ab9157 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T16:40:54.358Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T16:43:30.934Z"}
info: Connected to the database. {"timestamp":"2022-02-25T16:43:30.962Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T16:47:04.460Z"}
info: Connected to the database. {"timestamp":"2022-02-25T16:47:04.489Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T16:49:00.754Z"}
info: Connected to the database. {"timestamp":"2022-02-25T16:49:00.781Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ ID: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/6219088c7894e91d7c454332 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T16:53:58.933Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T16:54:15.960Z"}
info: Connected to the database. {"timestamp":"2022-02-25T16:54:15.989Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/6219088c7894e91d7c454332 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T16:54:26.783Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:02:22.536Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:02:22.563Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:07:15.966Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:07:15.996Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:09:00.658Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:09:00.745Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:10:01.989Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:10:02.044Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:26:08.616Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:26:08.672Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ ID: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:26:13.876Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:26:57.707Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:26:57.737Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:27:06.554Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:28:39.114Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:28:39.147Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:28:43.146Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:28:44.907Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:29:14.056Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:29:14.081Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'ID', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:29:18.171Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-25T17:30:11.770Z"}
info: Connected to the database. {"timestamp":"2022-02-25T17:30:11.801Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: '_id', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:30:19.686Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: '_id', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:30:21.776Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: '_id', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:30:22.688Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: '_id', UserID: 'UserID' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\anand\Desktop\fundoo-Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/note/62190bb92edb561ef0b01306 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-25T17:30:23.638Z"}
